west.yml

```diff
@@ -101,7 +101,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 46be8173d2453b16c617b555f96c59d62920a290
+      revision: d1330d29486ba8d86ea10e9eedae8ce5b39f5527
```

Comment:
Due to the order that changes merged in NXP's HAL, this PR is dependent on #44961. Essentially, updating the SHA of NXP's HAL to `d1330d29486ba8d86ea10e9eedae8ce5b39f5527` before #44961 merges would cause build failures on some of NXP's platforms, that #44961 contains the updates required to fix. I'm going to mark this PR as DNM rather than NAKing it, since there is no issue with the PR itself.

Reply:
@NickolasLapp I think we need to just remove the west.yml changes from this because the version in Zephyr now has a SHA that is past this SHA so we can't go backwards